ansible/vars/icds/icds_public.yml

```diff
@@ -3,19 +3,19 @@ CAS_SITE_HOST: 'cas.commcarehq.org'
 NO_WWW_SITE_HOST: 'icds-cas.gov.in'
 deploy_env: icds
 internal_domain_name: 'internal-icds.commcarehq.org'
-tableau_server: 10.247.24.11:80
+# is tableau necessary? tableau_server: 10.247.24.11:80
```

Comment:
I think that tableau isn't being used in the new cluster and the users will be using the new dasbhoard built by sol ddev

Reply:
going to keep it around for now just in case we do end up needing it